### PR TITLE
Add orientation marker to nav bar

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Added orientation markers to the current active section.

## QA
- Click through the site and see that the navigation is selected correctly

### CODE REVIEW

File: templates/main-nav.html [hunk 1]
```diff
@@ -95,7 +95,7 @@
         <li><a href="/desktop/whitepapers">Whitepapers</a></li>
       </ul>
     </li>
-    <li class="p-navigation__link">
+    <li class="p-navigation__link{%- if page_slug == 'topics' %} is-selected{% endif %}">
       <a href="#">Topics</a>
```
Comment (bartaz) [inline templates/main-nav.html]: Not really related to this PR, but 'Topics' is the only item in navigation that cannot be clicked which is a bit confusing.

**Comment by anthonydillon** [comment]
@bartaz yea, this is a known issue. I think it will be tackled by UX first.

**Comment by bartaz** [comment]
@anthonydillon You mean it's fine to merge it as it is and will be handled by UX separately?

**Comment by pmahnke** [comment]
@anthonydillon looks fine; however there is a conflict....

**Comment by anthonydillon** [comment]
Ouch, the entire file is conflicting. Will be easier to start this branch again. May wait for the development to die down a bit as I have been conflicted on this branch three times now.

**Pull request closed without merging**